### PR TITLE
[Doc-Improvement][Interchange-tab-locations][2140400]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/what-are-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/67a929901d8a7467954cbceccfcee9853c7d2a6d/msteams-platform/tabs/what-are-tabs.md) | 88 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d6e239ec-5c09-460f-8e6d-ce7da11901cd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/what-are-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f76dbf1407e2dc0ce66677c6f8ca13f9c3f9898d/msteams-platform/tabs/what-are-tabs.md) | 88 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0877f2be-5b46-4485-a353-3fd0501a1960) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/what-are-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/63db77fc6ad039f3d7d5a1fedf86e0883362bbf8/msteams-platform/tabs/what-are-tabs.md) | 88 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/af69df8d-c4e7-48de-93df-35c71cf2c54d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/what-are-tabs.md [hunk 1]
```diff
@@ -28,7 +28,10 @@ There are two types of tabs available in Teams, personal and channel or group. [
...
+You can build tabs with Adaptive Cards and centralize all Teams app capabilities by eliminating the need for a different backend for your bots and tabs. [Stage View](~/tabs/tabs-link-unfurling.md) is a new UI component that allows you to render the content opened in full screen in Teams and pinned as a tab. The existing [link unfurling](~/tabs/tabs-link-unfurling.md) service is updated, so that it is used to turn URLs into a tab using an Adaptive Card and Chat Services. You can [create conversational tabs](~/tabs/how-to/conversational-tabs.md) using conversational sub-entities that allow users to have conversations about sub-entities in your tab, such as specific task, patient, and sales opportunity, instead of discussing the entire tab. You can make changes to [tab margins](~/resources/removing-tab-margins.md) to enhance the developer's experience when building apps. You can drag the tab and place it in the desired position to interchange the tab positions within your personal apps and channel or group chats. 
+
+> [!NOTE]
+> Posts, Files, and Chat tabs can't be moved from their positions.
```
Comment (surbhigupta12) [inline msteams-platform/tabs/what-are-tabs.md:34]: Change to:

> **Posts**, **Files**, and **Chat** tabs can't be moved from their positions.

**Review by v-habibaumme — DISMISSED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/what-are-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/69940aff526333ceef0d9294b706054d5fa8fd34/msteams-platform/tabs/what-are-tabs.md) | 88 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/619d9ee5-7a0b-4917-b957-3b2ecc201e4e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Review by v-habibaumme — COMMENTED** [review]
Approved

**Pull request merged**